lib/tmuxinator/cli.rb

```diff
@@ -48,64 +56,93 @@ def completions(arg)
     map "o" => :new
     map "e" => :new
     map "n" => :new
+    method_option :local, type: :boolean,
+                          aliases: ["-l"],
+                          desc: "Create local project file at ./.tmuxinator.yml"
 
     def new(name)
-      config = Tmuxinator::Config.project(name)
-
-      unless Tmuxinator::Config.exists?(name)
-        template = Tmuxinator::Config.default? ? Tmuxinator::Config.default : Tmuxinator::Config.sample
-        erb  = Erubis::Eruby.new(File.read(template)).result(binding)
-        File.open(config, "w") { |f| f.write(erb) }
+      project_file = if options[:local]
+                       Tmuxinator::Config::LOCAL_DEFAULT
+                     else
+                       Tmuxinator::Config.default_project(name)
+                     end
+      unless File.exists?(project_file)
```

Comment:
This should be `Tmuxinator::Config.exists?` otherwise this code block isn't entered during the tests. Not sure why travis is passing as I'm getting a failure locally here.